brands/ghostery/branding/pref/firefox-branding.js

```diff
@@ -182,3 +182,7 @@ pref("browser.newtabpage.activity-stream.section.highlights.includeBookmarks", f
 pref("browser.newtabpage.activity-stream.section.highlights.includeDownloads", false);
 pref("browser.newtabpage.activity-stream.section.highlights.includePocket", false);
 pref("browser.newtabpage.activity-stream.section.highlights.includeVisited", false);
+
+// Number of usages of the web console.
+// If this is less than 5, then pasting code into the web console is disabled
+pref("devtools.selfxss.count", 5);
```

Comment:
It looks to me like setting it to `pref("devtools.selfxss.count", 0);` as the original branding does also works (as soon as you enter a command into the console it changes to 5, should we instead set the default to 0?

Reply:
lets do that